File: 1848. Minimum Distance to the Target Element.php
class Solution {
    function getMinDistance($nums, $target, $start) {
    $arr = [];
    for ($i = 0; $i < count($nums); $i++) { if ($nums[$i] == $target) {array_push($arr, abs($i - $start)); }}
    return min($arr);
    }
}

File: 1848. Minimum Distance to the Target Element.rs
impl Solution {
    pub fn get_min_distance(nums: Vec<i32>, target: i32, start: i32) -> i32 {
    let mut arr: Vec<i32> = Vec::new();
    for i in 0..nums.len() {
        if nums[i] == target {
            arr.push((i as i32 - start).abs());
        }
    }
    *arr.iter().min().unwrap_or(&0)
    }
}
